dockerfile

```diff
@@ -17,6 +17,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     unixodbc-dev \
     unzip \
     wget \
+    postgresql-client \ 
```

Comment:
@rephus can you move this to only be included in the `dev` stage? Otherwise it's unnecessarily in the production container